docker-compose.yml

```diff
@@ -37,6 +37,8 @@ services:
   frontend:
     image: "${DOCKER_REPO}corpora-frontend"
     platform: linux/amd64
+    profiles:
```

Comment:
What does adding profile do?

Reply:
It allows us to leverage happy slicing, it depends on these profile annotations